roles/archive/templates/etc/cron.weekly/count_archive_hits

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#set -e
+
+log_root={{ varnish_logroot|default('/var/www/varnish_logs') }}
+log_user={{ varnish_user|default('varnishlog') }}
+log_group={{ varnish_group|default('varnish') }}
+combined_out="${log_root}/varnishncsa_combined.log"
+current_datestamp=$(date -d "$(printf '%(%m/%d/%Y)T 00:00:00')" +%s)
```

Comment:
date can do that by itself: `date -d '00:00:00' +%s` - you can add "today" if you want to make it more readable: `date -d 'today 00:00:00' +%s`.